templates/exadata-limits/main.tf

```diff
@@ -0,0 +1,24 @@
+locals {
+
```

Comment:
Can we use template file instead the inline formatting https://developer.hashicorp.com/terraform/language/functions/templatefile